src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -2115,8 +2115,8 @@ f1_incm_stmnt_2,69,1994,Net Extraordinary Items (Enter Total of line 67 less lin
 f1_incm_stmnt_2,70,1994,Income Taxes-Federal and Other (409.3),reported_value,income_taxes_extraordinary_items,,
 f1_incm_stmnt_2,71,1994,Extraordinary Items After Taxes (Enter Total of line 69 less line 70),calculated_value,extraordinary_items_after_taxes,,
 f1_incm_stmnt_2,72,1994,Net Income (Enter Total of lines 65 and 71),calculated_value,net_income_loss,,
-f1_incm_stmnt_2,25,2003,,header,#N/A,This table starts at row 27 after 2003
-f1_incm_stmnt_2,26,2003,,header,#N/A,This table starts at row 27 after 2003
+f1_incm_stmnt_2,25,2003,,header,#N/A,,This table starts at row 27 after 2003
+f1_incm_stmnt_2,26,2003,,header,#N/A,,This table starts at row 27 after 2003
```

Comment:
Just changed the column that your notes were in (they were in the `dbf_only` column before)